### ISSUE TITLE
Make feature expressions read-only when used

### ISSUE THREAD

**Issue by Kogoro** [tthuem/VariantSync#36]
Changing a feature expression which is already in use leads to some unresolved questions.

- Update all markers and mappings when some feature expression changes?
- ...

For the time being, the feature expressions should be read-only when in use.

**Comment by pmbittner** [comment]
Yes, feature expressions should be readonly. If you want another feature expression you would "create a new one".

**Comment by tthuem** [comment]
We should try to stick to our terminology also in issues. What is meant by a feature expression here? Feature mapping? Will changes be mapped to both, feature context and feature mapping? Could be useful for synchronization purposes.

**Comment by pmbittner** [comment]
In my understanding, the feature expressions mentioned here are those that you can choose from for the feature context (i.e., those in the ComboBox). Changing them neither changes any existing feature mappings (which it also should not) and should usually also not affect the current feature context.
